tap_s3_csv/transform.py

```diff
@@ -184,6 +183,33 @@ def transform_recur(self, data, schema, path, source_type=None):
                 Error(path, data, schema, logging_level=LOGGER.level))
             return False, None
 
+    def transform_schema_recur(self, schema):
```

Comment:
why is this a method inside class Transformer? it doesn't even use anything in the class. might as well be just a method like resolve_filter_fields